src/components/card/hotels.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
```

Comment:
I expected to see card.js file or index.js, but not hotels.js inside card folder.

Reply:
corrected